src/Bicep.Core/Analyzers/Linter/LinterAnalyzer.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Deployments.Core.Extensions;
+using Bicep.Core.Analyzers.Interfaces;
+using Bicep.Core.Configuration;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Parsing;
+using Bicep.Core.Semantics;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+
+namespace Bicep.Core.Analyzers.Linter
+{
+    internal class LinterAnalyzer : IBicepAnalyzer
+    {
+        public const string SettingsRoot = "analyzers";
+        public const string AnalyzerName = "bicepcorelinter";
```

Comment:
>bicepcorelinter [](http://example.com/codeflow?start=44&length=15)

Should we simplify the name here? `core` or just `bicep` maybe?

Reply:
decision: core

Reply:
changed bicepcorelinter to 'core'